app.js

```diff
@@ -0,0 +1,28 @@
+// App should require config and logger.
+const http = require('http');
+const config = require("./config")
+const logger = require("./logger")
+
+// Env should be passed through as cli argument
+const cliOptions = []
+const myArgs = process.argv.slice(2);
+
+myArgs.map((arg) => {
```

Comment:
pls move this logic inside config and hide it from here

Reply:
Updated.